src/features/training-program/training-program.tsx

```diff
@@ -17,20 +17,23 @@ export const TrainingProgram = ({ courseName, type }: TrainingProgramProps) => {
     courseName.includes('badge') ? 'aws fundamentals' : courseName,
     type,
   );
-
   const course = data as Course;
 
   const { title, content, image } = contentMap[courseName];
 
   const buttonLabel = courseName === 'react ru' ? 'Записаться' : 'Register';
 
+  const filteredContent =
+    type === 'Pre-school'
+      ? content.filter((component) => component.type.name !== 'Subtitle')
+      : content;
   return (
     <section className="training-program container">
       <div className="training-program content column-2">
         <div className="left">
           <Title text={title} hasAsterisk />
 
-          {content.map((component, index) => cloneElement(component, { key: index }))}
+          {filteredContent.map((component, index) => cloneElement(component, { key: index }))}
 
           <Button label={buttonLabel} href={course?.enroll} />
```

Comment:
Button with href?

Reply:
As I see - button component contains anchor tag and leads to registration for course.